m3u8/model.py

```diff
@@ -629,7 +629,7 @@ def dumps(self, last_segment, timespec="milliseconds"):
 
             output.append(
```

Comment:
I think we should preserve what the what we receive as input.

Meaning, we should expand [the parser to include something](https://github.com/globocom/m3u8/blob/master/m3u8/parser.py#L123) like `cue_out_duration_property` to true.

```python
        elif line.startswith(protocol.ext_x_cue_out):
            _parse_cueout(line, state)
            state["cue_out_start"] = True
            state["cue_out"] = True
            if line.contains(somethingsomething):
                   state["cue_out_explicitly_duration"] = True

```

And then here, we would need to check to see if we'd need to add the Duration or not. WDYT?

Reply:
Otherwise we'll transform manifests without Duration to have one.

Reply:
Yeah, that's true. I changed it and adapted for parsing it only when DURATION is available. Also, a new test has been created and some code has been refactored regarding the new state.